english/forest/scream/sing/sing.md

```diff
@@ -0,0 +1,23 @@
+You decide to sing to the tiger. One last ballad, as an ode to the life you've lived. Your last gift to the world, although no one but the tiger will hear your angelic voice.
+
+The tiger is startled by this sudden turn of events. He sits down and listens to your song.
+
+"Oh sir tiger, I know you must take my life"
+"Please let it be over swiftly, and eat me with your jaws"
+"If only you could find it in your heart to return me to my wife"
+"Then I will forever more kiss your mighty paws"
+
+The tiger seems moved by the depth of your lyrics. You continue.
+
+"Oh my dear friend, I wish our paths had not crossed"
+"If you would let me go, I will -"
+
+The tiger stopped you mid-sentence.
+
+"I am so moved by your words of song," the tiger said. "I shall let you go, on one condition."
+
+"What is it?" you replied.
+
+<scene cut>
```

Comment:
It looks like this is the line that is causing Casey trouble. I should fix her, but you should also probably remove the angle brackets in this line, since it's getting interpreted as an html tag. When I view the file on GitHub, this line does not appear for that reason. (If you want, you could use asterisks to put this in italics - that is, use `*scene cut*` instead of `<scene cut>`.)